pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -126,6 +132,8 @@ type desiredStateOfWorldPopulator struct {
 	hasAddedPodsLock          sync.RWMutex
 	csiMigratedPluginManager  csimigration.PluginManager
 	intreeToCSITranslator     csimigration.InTreeToCSITranslator
+	pvcsUnderDeletion         map[string]bool
+	numClientGets             int64
```

Comment:
I believe mock k8s client used in unit testing should already have functionality like this.

Reply:
In the existing test, I only see podtest.NewMockCheckpointManager, not the client.